src/main/java/roomescape/controller/TimeController.java

```diff
@@ -0,0 +1,53 @@
+package roomescape.controller;
+
+import java.util.List;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+import roomescape.dto.TimeDto;
+import roomescape.dto.TimeResDto;
+import roomescape.service.TimeService;
+
+@RestController
+@RequestMapping("/times")
+public class TimeController {
+    private final TimeService timeService;
+
+    public TimeController(TimeService timeService) {
+        this.timeService = timeService;
+    }
+
+    @PostMapping
+    @ResponseBody
+    public ResponseEntity<TimeResDto> addTime(@RequestBody TimeDto timeDto) {
+        TimeResDto timeResDto = timeService.addTime(timeDto);
+        HttpHeaders headers = createHeader("/times/" + timeResDto.id());
+        return new ResponseEntity<>(timeResDto, headers, HttpStatus.CREATED);
+    }
```

Comment:
```suggestion
@RestController // <--
@RequestMapping("/times")
public class TimeController {
    private final TimeService timeService;

    public TimeController(TimeService timeService) {
        this.timeService = timeService;
    }

    @PostMapping
    public ResponseEntity<TimeResDto> addTime(@RequestBody TimeDto timeDto) {
        TimeResDto timeResDto = timeService.addTime(timeDto);
        HttpHeaders headers = createHeader("/times/" + timeResDto.id());
        return new ResponseEntity<>(timeResDto, headers, HttpStatus.CREATED);
    }
```

<img width="293" alt="image" src="https://github.com/user-attachments/assets/7a527523-d3be-4fd6-b4e0-c21573f7ade1">

- `@RestController` 어노테이션 내부를 살펴보면 `@ResponseBody`가 존재하기 때문에 지우면 더 좋을 것 같아요!

Reply:
앗... 깜박했네요 감사합니다...😅